src/hasReachedStyle.js

```diff
@@ -1,4 +1,8 @@
-export default function hasReachedStyle(currentStyle, style) {
+/* @flow */
+import type {Style} from './Types.js';
+
+export default function hasReachedStyle(currentStyle: Style,
+                                        style: Style): boolean {
```

Comment:
I was expecting this indentation to be fixed everywhere not only in the place I left my comment ;)